packages/framework/fluid-framework/src/index.ts

```diff
@@ -22,85 +22,75 @@ export type {
 	IFluidContainer,
 	IFluidContainerEvents,
 	IMember,
-	IRootDataObject,
 	IServiceAudience,
 	IServiceAudienceEvents,
 	LoadableObjectClass,
 	LoadableObjectClassRecord,
 	LoadableObjectCtor,
-	LoadableObjectRecord,
 	MemberChangedListener,
 	SharedObjectClass,
 } from "@fluidframework/fluid-static";
+export type { ISharedMap, ISharedMapEvents, IValueChanged } from "@fluidframework/map";
+export { SharedMap } from "@fluidframework/map";
+
 export type {
-	ICreateInfo,
-	IDirectory,
-	IDirectoryClearOperation,
-	IDirectoryCreateSubDirectoryOperation,
-	IDirectoryDataObject,
-	IDirectoryDeleteOperation,
-	IDirectoryDeleteSubDirectoryOperation,
-	IDirectoryEvents,
-	IDirectoryKeyOperation,
-	IDirectoryNewStorageFormat,
-	IDirectoryOperation,
-	IDirectorySetOperation,
-	IDirectoryStorageOperation,
-	IDirectorySubDirectoryOperation,
-	IDirectoryValueChanged,
-	ILocalValue,
-	ISerializableValue,
-	ISerializedValue,
-	ISharedDirectory,
-	ISharedDirectoryEvents,
-	ISharedMap,
-	ISharedMapEvents,
-	IValueChanged,
-} from "@fluidframework/map";
-export {
-	DirectoryFactory,
-	LocalValueMaker,
-	MapFactory,
-	SharedDirectory,
-	SharedMap,
-} from "@fluidframework/map";
-export type {
-	DeserializeCallback,
-	IInterval,
-	IIntervalCollectionEvent,
-	IIntervalHelpers,
-	IJSONRunSegment,
-	IMapMessageLocalMetadata,
-	IIntervalCollection,
-	IntervalLocator,
-	ISequenceDeltaRange,
-	ISerializableInterval,
-	ISerializedInterval,
-	ISharedIntervalCollection,
-	ISharedSegmentSequenceEvents,
-	ISharedString,
-	IValueOpEmitter,
-	SerializedIntervalDelta,
-	SharedStringSegment,
-} from "@fluidframework/sequence";
+	AllowedTypes,
+	ApplyKind,
+	ArrayToUnion,
+	CheckoutEvents,
+	Events,
+	ExtractItemType,
+	FlexList,
+	FlexListToUnion,
+	IDisposable,
+	ImplicitAllowedTypes,
+	ImplicitFieldSchema,
+	InsertableObjectFromSchemaRecord,
+	InsertableTreeFieldFromImplicitField,
+	InsertableTreeNodeFromImplicitAllowedTypes,
+	InsertableTypedNode,
+	IsEvent,
+	ISubscribable,
+	ITree,
+	LazyItem,
+	MakeNominal,
+	NodeBuilderData,
+	NodeFromSchema,
+	ObjectFromSchemaRecord,
+	RestrictiveReadonlyRecord,
+	Revertible,
+	TreeApi,
+	TreeArrayNodeBase,
+	TreeFieldFromImplicitField,
+	TreeLeafValue,
+	TreeMapNode,
+	TreeMapNodeBase,
```

Comment:
These base types only exist because of the need to share logic between the now removed simple-tree-types and the class tree ones. We should be able to remove them by merging the API/types together.